### PR TITLE
Add dbanimes

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**
Tested, but not extensively. Some episodes only use unsupported servers, which I'll maybe try to support later (like clipwatching - of which I could find no working link, or jetload - which uses invisible Google recaptcha to generate a token that it then posts along with the video id to jetload.net/jet_secure, or vidfast - which simply wouldn't play for me, rip)

### CODE REVIEW

File: anime_downloader/config.py [hunk 1]
```diff
@@ -128,6 +128,10 @@
                 "gounlimited"
             ]
         },
+        'dbanimes': {
+            'server': 'fembed',
+            'fallback_servers': ['mixdrop', 'sendvid']
```
Comment (Blatzar) [inline anime_downloader/config.py:133]: I suggest using sort_sources instead, or at least in the future for less headaches with selected servers and all that.

**Comment by nate-moo** [comment]
`426: Upgrade Required`

**Comment by Blatzar** [comment]
Going to fix the rest

**Comment by Blatzar** [comment]
TODO: Sources found on https://dbanimes.com/your-name-vostfr/ can probably get added

**Pull request merged**